plugins/inputs/eventhub/eventhub.go

```diff
@@ -0,0 +1,382 @@
+package eventhub
+
+// TODO: (optional) Test authentication with AAD TokenProvider environment variables?
+// TODO: (optional) Event Processor Host, only applicable for multiple Telegraf instances?
```

Comment:
This is an interesting idea, it seems like it would be a separate program above Telegraf, generating new telegraf.conf files and reloading Telegraf as needed.  Telegraf would communicate the offsets back to it.

We can think about this later of course, for now can you go ahead and remove any TODO messages in the pr.

Reply:
Moved to README